README.md

```diff
@@ -4,16 +4,107 @@
 [![Build Status](https://travis-ci.org/jboss-fuse/vscode-datavirt.svg?branch=master)](https://travis-ci.org/jboss-fuse/vscode-datavirt)
 [![License](https://img.shields.io/badge/license-Apache%202-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+## Prerequisites
+
+- recent Visual Studio Code
+
+
 ## Features
 
-...
+Red Hat Integration - Data Virtualization Tooling enables you to create a Virtual Database definition file (VDB) and to visualize and manipulate its contents inside [VS Code](https://code.visualstudio.com/).
 
+### The Data Virtualization view
+The extension provides its own view to [VS Code](https://code.visualstudio.com/) which is called `Data Virtualization`.
```

Comment:
official terminology is "Tree View" and not only "view" https://code.visualstudio.com/api/extension-capabilities/extending-workbench
but not sure if it is worthy to be that specific

Reply:
fine to use non-official but widely in other extensions the "view" terminology. See Brian comment https://github.com/jboss-fuse/vscode-datavirt/pull/143#issuecomment-620615556